Ginger/GingerCoreCommon/Repository/NewRepositorySerializer.cs

```diff
@@ -681,7 +685,10 @@ private static object xmlReadObject(Object Parent, XmlReader xdr, RepositoryItem
                     obj = targetObj;
                 }             
 
-
+                if (string.IsNullOrEmpty(filePath) == false && obj != null && obj.GetType().IsSubclassOf(typeof(RepositoryItemBase)))
```

Comment:
Codacy found an issue: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5848147)